DuckDuckGo/Updates/UpdateController.swift

```diff
@@ -182,6 +182,11 @@ final class UpdateController: NSObject, UpdateControllerProtocol {
 
         updater = SPUUpdater(hostBundle: Bundle.main, applicationBundle: Bundle.main, userDriver: userDriver, delegate: self)
 
+        // We don't want SUAutomaticallyUpdate enabled because it interferes with our custom updater UI
+        if updater?.automaticallyDownloadsUpdates == true {
```

Comment:
Prior to the Sparkle custom UI implementation, `automaticallyDownloadsUpdates` is set to `true` and the corresponding `SUAutomaticallyUpdate` property will get stuck in UserDefaults, interfering with our custom updater UI. 

This fix removes the setting from UserDefaults and unblocks further updates.